editor/CHANGES

```diff
@@ -1,19 +1,12 @@
 [0.6.0] ~
-- [BREAKING CHANGE] Migrated from Kryo to JSON for .registry and .pro files. Migration handled automatically.
```

Comment:
Why did you remove these lines?

Reply:
I apologize. Don't know how I managed to do that.

Reply:
Hm I know you pushed a commit to bring that line back, but its not just that line, other lines also removed click on "files changed" in on this PR and you will see other lines in the Changes file removed that should not be.